iati/iati/custom_middleware.py

```diff
@@ -0,0 +1,106 @@
+"""Middleware for redirecting mixed case urls into lowercase."""
+from django import http
+from django.conf import settings
+from django.utils.functional import cached_property
+
+
+class RedirectIATISites:
```

Comment:
@Ocre42 

I realised that having a middleware that did two things wasn't best practise - redirects to external sites are now handled here